client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/jetpack-search.tsx

```diff
@@ -5,16 +5,13 @@ import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { useSelector } from 'calypso/state';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import ThankYouJetpackSearchProduct from '../products/jetpack-search-product';
-import type { ThankYouJetpackSearchProductProps } from '../products/jetpack-search-product';
 import type { ReceiptPurchase } from 'calypso/state/receipts/types';
 
 export type JetpackSearchThankYouProps = {
 	purchase: ReceiptPurchase;
 };
 
-export default function JetpackSearchThankYou( {
-	purchase,
-}: Omit< ThankYouJetpackSearchProductProps, 'siteId' > ) {
```

Comment:
@chad1008 I noticed you made this change https://github.com/Automattic/wp-calypso/pull/88262/commits/b6275c8ebca4741b1e3dfa53d456bf52ddcf1ff0 to resolve TS error. The initial problem was actually the fact that I supplied the wrong type 😓 It should be fixed by https://github.com/Automattic/wp-calypso/pull/88262/commits/9b931a2b201e0a874be1e6dcaca8584de0c7670b

Reply:
Ah, cool. The updated types look good to me 😄